README.md

```diff
@@ -222,6 +224,17 @@ The tag with the higher version is used.
 
 MinVer will use the height on the first path followed where the history diverges. The paths are followed in the same order that the parents of the commit are stored in git. The first parent is the commit on the branch that was the current branch when the merge was performed. The remaining parents are stored in the order that their branches were specified in the merge command.
 
+### Can I increment other parts of the version besides `patch`
+
+Yes! MinVer offers both a cli (`-i | --increment-segment <SEGMENT>`) option and an MSBuild property (`MinVerAutoIncrement`) to override the default behavior. The default behavior in MinVer defines, that for non-tagged commits, the `alpha.0` [suffix and the height will be appended to the version](#how-it-works). But it is possible to increment other segments of the version like the major and minor. The default for these parameters are set to `patch` and represent the default behavior. To increment the minor the value can be set to `minor` and for the major to `major`.
+
+```
+latest version    MinVerAutoIncrement   at height     resolves to
+1.2.3+a           major                 1             2.0.0-alpha.0.1

```

Comment:
These cases are not in the test in `BuildMetadata.PreviousTag` anymore! Just the default case (patch) is represented there!

Reply:
That's true, because the `BuildMetadata` class is for testing the build metadata feature. We should add a new class for testing the auto incrementing feature, which isolates test cases each of the `VersionPart` values.